mockup_package/mockup/image_generator.py

```diff
@@ -38,6 +38,21 @@ def __init__(self, original_img_path, phone_slug, device_info):
     def create_fit_resolution_image(self):
         device_ratio = self.d_size[0] / self.d_size[1]
         image = Image.open(self.original_img_path).convert("RGBA")
+
+        # Solve image orientation issue # noqa: E501
+        # ref: https://stackoverflow.com/questions/56854176/is-there-a-way-i-can-detect-the-image-orientation-and-rotate-the-image-to-the-ri # noqa: E501
+        for orientation in ExifTags.TAGS.keys():
+            if ExifTags.TAGS[orientation] == "Orientation":
+                break
```

Comment:
I read the stackoverflow thread and did not find any answers implementing your logic here. Can you 

1. point the link to the SO answer you adopted?
2. maybe add some more context as python comments? like
    - what was the problem?
    - what was the fix?

Reply:
@pkong-ds sor I put incorrect link, should be this one: https://stackoverflow.com/questions/13872331/rotating-an-image-with-orientation-specified-in-exif-using-python-without-pil-in
updated and added more context